### PR TITLE
Cost info operator

### PR DESCRIPTION
**Pull request by braktar**

### CODE REVIEW

File: lib/interpreters/compute_several_solutions.rb [hunk 1]
```diff
@@ -172,7 +172,7 @@ def self.find_best_heuristic(service_vrp)
 
         custom_heuristics << 'supplied_initial_routes' if vrp.routes.any?
 
-        times = []
+        elpased_array = []
```
Comment (senhalil) [inline lib/interpreters/compute_several_solutions.rb:175]: there is something wrong here :thinking: I remember this exact modification from another PR -- which suggested `elapsed_array` that became `elapsed_times` but here is `elpased_array` again with a typo?

Comment (senhalil) [reply lib/interpreters/compute_several_solutions.rb:175]: @braktar do you understand what happened? Am I hallucinating :confused:

Comment (braktar) [reply lib/interpreters/compute_several_solutions.rb]: It comes with this PR https://github.com/Mapotempo/optimizer-api/pull/359 which not merged yet

Comment (senhalil) [reply lib/interpreters/compute_several_solutions.rb]: Did you make overlapping modification in two different commits of two different PRs :sweat: no wonder I am confused -- it was a real deja-vu

Comment (braktar) [reply lib/interpreters/compute_several_solutions.rb]: This PR requires #359 so yeah it overlaps

Comment (senhalil) [reply lib/interpreters/compute_several_solutions.rb]: I just realized the commit names !

Comment (senhalil) [reply lib/interpreters/compute_several_solutions.rb]: https://github.com/Mapotempo/optimizer-api/pull/359 is merged you can rebase this one :t-rex:

**Review by senhalil — APPROVED** [review]

**Pull request merged**